lib/node_modules/@stdlib/stats/base/dcumax/src/main.c

```diff
@@ -30,26 +31,35 @@
 * @param Y        output array
 * @param strideY  Y stride length
 */
-void stdlib_strided_dcumax( const int64_t N, const double *X, const int64_t strideX, double *Y, const int64_t strideY ) {
+void API_SUFFIX( stdlib_strided_dcumax ) ( const CBLAS_INT N, const double *X, const CBLAS_INT strideX, double *Y, const CBLAS_INT strideY ) {
```

Comment:
```suggestion
void API_SUFFIX(stdlib_strided_dcumax)( const CBLAS_INT N, const double *X, const CBLAS_INT strideX, double *Y, const CBLAS_INT strideY ) {
```

Exception to the rule due to conventions elsewhere.